_includes/assets/js/indicatorModel.js

```diff
@@ -191,8 +191,13 @@ var indicatorModel = function (options) {
   }());
 
   var headlineColor = '777777';
-  var colors = ['7e984f', '8d73ca', 'aaa533', 'c65b8a', '4aac8d', 'c95f44'];
-
+  
+  // use custom colors if defined
+  this.colorSet = {{ site.graph_color_set | jsonify }};
+  this.numberOfColors = {{ site.graph_color_number | jsonify }};
+  this.customColors = {{ site.graph_color_list | jsonify }};
```

Comment:
Since `colorSet`, `numberOfColors`, and `customColors` are not needed anywhere else in this script, what about moving them to `chartColors.js`?

Reply:
Good point but how do I call the values from `config.yml` from `chartColors.js`? 
`this.colorSet = {{ site.graph_color_set | jsonify }}; ` seems not to work here.

Reply:
@brockfanning ^^^^

Reply:
@MoBosse I don't think there's anything special about indicatorModel.js, so if it works there then it should also work in chartColors.js. Maybe something like this at the top of the function?
```
var colorSet = {{ site.graph_color_set | jsonify }},
    numberOfColors = {{ site.graph_color_number | jsonify }},
    customColorList = {{ site.graph_color_list | jsonify }};
```

Reply:
@brockfanning thats pretty much what I tried [here](https://github.com/Nachhaltige-Entwicklung-Deutschland/open-sdg-site-starter/blob/develop/assets/js/chartColors.js#L3) but it keeps giving me a console-message that says 

_SyntaxError: expected property name, got '{'_

Reply:
I tested this in a Version of chartColors.js that is saved in the assets folder of the site-starter repo. In the includes folder it seems to work. Sorry for the confusion.